examples/get-started/pure-js/mapbox/webpack.config.js

```diff
@@ -16,10 +16,7 @@ const CONFIG = {
       // From mapbox-gl-js README. Required for non-browserify bundlers (e.g. webpack):
       'mapbox-gl$': resolve('./node_modules/mapbox-gl/dist/mapbox-gl.js')
     }
-  },
-
-  // Optional: Enables reading mapbox token from environment variable
-  plugins: [new webpack.EnvironmentPlugin(['MapboxAccessToken'])]
```

Comment:
If you remove this, you also need to remove the previous require of webpack

Reply:
You can use the linter to see the list of files with this issue

Reply:
fixed: dbebf57